packages/terminal/src/node/inversify.spec-config.ts

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
According to https://github.com/theia-ide/theia/blob/master/doc/Testing.md#test-directory-structure this should go to `packages/terminal/src/node/test/inversify.spec-config.ts`

Reply:
Right, fixing